spec/models/contact_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  let(:user) { User.create(name: 'User Name', age: 20, biography: 'bio') }
+  let(:user_02) { User.create(name: 'User_02 Name', age: 30, biography: 'bio') }
+  let(:contact) { Contact.create(name: 'Contact Name', birthday: '01/01/2021', user_id: user.id) }
+
+  it 'is valid with valid name attribute' do
+    expect(contact).to be_valid
+  end
+
+  it 'is not valid when two contacts with same name in same user' do
+    contact
+    expect(Contact.create(name: 'Contact Name', user_id: user.id)).to_not be_valid
```

Comment:
Ao invés de usar o matcher `be_valid` não seria melhor testar a razão do contact estar inválido? Se não isso pode ocasionar em falsos positivos

Reply:
entendi, vou mudar os testes nesse sentido, seria um `expect(contact).to raise_error (Nome do ero)` ?